lib/Request.js

```diff
@@ -229,11 +229,14 @@ Request.prototype.status = function(code, body) {
     }
 }
 
-Request.prototype.error = function(httpStatusCode, message, errorCode, errorString, propertyErrors) {
+Request.prototype.error = function(httpStatusCode, errorObject, propertyErrors) {
```

Comment:
Since this is a breaking change for the framework, let's bump the `major` version number when publishing.